### ISSUE TITLE
How to install without a package manager?

### ISSUE THREAD

**Issue by philgoetz** [gabrielelana/vim-markdown#96]
The README file says this:

> If you are not using a package manager, download the [tarball](https://github.com/gabrielelana/vim-markdown/archive/master.tar.gz) and do this:
> 
> ```
> $ cp vim-markdown-master.tar.gz ~/.vim
> $ cd ~/.vim
> $ tar --strip-components=1 --overwrite -zxf vim-markdown-master.tar.gz
> $ rm vim-markdown-master.tar.gz
> ```
But it looks like this replaces my .vimrc with the one from the tar!
Obviously I can't do that.
Also, I tried it anyway, and it has no effect on my vim.
(When I tried it, I first created the directory ~/.vim, under the assumption that you wanted to copy vim-markdown-master.tar.gz to the file ~/vim-markdown-master.tar.gz, not to the file ~/.vim. )

How can I install this plugin without a package manager, and without wiping all my other customizations of vim?
Can I just insert all the lines of the given .vimrc into my .vimrc (or _vimrc)?

**Comment by neingeist** [comment]
Vim 8 has a native way to load something like this, see for example: https://learnvim.irian.to/customize/vim_packages

I still just use pathogen instead (for the occasional old Vim and because I'm too lazy to change) and put plugins into `.vim/bundle`.

**Comment by philgoetz** [comment]
Thanks. My vim has packages, but I don't know if I can use them, as there is no .vim directory. It's gvim, for Windows, which is an official release, so it should have some way to use packages; but it isn't documented. I tried using the vim folder instead of the .vim folder, but it doesn't seem to work.

But after trying to figure out whether my attempt at installing it had worked, I realized that I have no idea how to use Markdown. I just wanted to be able to make folding and unfolding nested text lists. But the documentation for this plugin makes no sense to me. I can't tell what it's talking about, how to invoke markdown mode, how to make markdown lists, or what the plugin actually does. I give up.

BTW, Vim 9.0 is available now.
